lib/foreman/renderer/scope/macros/transpilers.rb

```diff
@@ -0,0 +1,38 @@
+module Foreman
+  module Renderer
+    module Scope
+      module Macros
+        module Transpilers
+          extend ApipieDSL::Module
+
+          apipie :method, "Calls ct transpiler, path and arguments configurable in settings" do
+            required :input, String, desc: 'input YAML'
+            # optional :validate_input, Boolean, desc: 'validate YAML input and throw parsing error (defaults true)'
+            # optional :validate_output, Boolean, desc: 'validate JSON input and throw parsing error (defaults false)'
```

Comment:
Don't know how to write boolean doc, @ofedoren

Reply:
Sorry for the delay. Please, try `optional :validate_input, one_of: [true, false], desc: ...`